metadata_service/proxy/neo4j_proxy.py

```diff
@@ -821,6 +821,38 @@ def _build_user_from_record(record: dict, manager_name: str = '') -> UserEntity:
                           role_name=record.get('role_name'),
                           manager_fullname=manager_name)
 
+    @staticmethod
+    def _get_user_resource_relationship_clause(relation_type: UserResourceRel, id: str = None,
+                                               user_key: str = None,
+                                               resource_type: ResourceType = ResourceType.Table) -> str:
+        """
+        Returns the relationship clause of a cypher query between users and tables
```

Comment:
between users and resources?

Reply:
I know this is a private method, but do you think it would be helpful to have a unit test for this to help understand input/output and its translation.